.github/workflows/build-images.yaml

```diff
@@ -115,15 +115,12 @@ jobs:
           sudo apt-get install -y qemu-user-static
       - name: Run make bundle (main)
         if: ${{ github.ref_name == env.MAIN_BRANCH_NAME }}
-        run: make bundle REGISTRY=${{ env.IMG_REGISTRY_HOST }} ORG=${{ env.IMG_REGISTRY_ORG }}
+        run: make bundle REGISTRY=${{ env.IMG_REGISTRY_HOST }} ORG=${{ env.IMG_REGISTRY_ORG }} IMAGE_TAG=${{ github.sha }}
```

Comment:
We can remove the _SHA_ tag from main branch, since it's not needed and causing the failing bundle and manifest verification. With the branch ones it will do, since the job collecting the sha tags from quay only cares for the latest one:
https://github.com/Kuadrant/kuadrant-operator/blob/main/.github/workflows/update-stored-dependencies-version.yaml
Then the scheduled event will create kuadrant bundle and catalog based on those here:
https://github.com/Kuadrant/kuadrant-operator/blob/main/.github/workflows/build-images-scheduled.yaml

Reply:
I'm not quite sure I follow here - if we remove the `IMAGE_TAG` from the `main` flow all catalog images will contain the bundle tagged as `latest` and not the specific commit sha.

Reply:
In the end, the sha will need to be for main only, and fix the verification stage. We were relying on the generation of the latest sha pushed to quay (from branches) to generate the kuadrant-operator catalog in a scheduled way for automated testing needed for QE (the workflows shared in my first comment). After some thought, given the restrictions on the before mentioned implementation, it needs to be changed to only main. This would be the main issue https://github.com/Kuadrant/kuadrant-operator/issues/227.

Reply:
Sorry for the confusion generated and the lack of information in the original issue.